### ISSUE TITLE
Effect Missing In Latest Forge - Also Latest Better Loading Screen

### ISSUE THREAD

**Issue by DPxKnightxZero** [AlexIIL/BetterLoadingScreen#18]
[Client]-BetterLoadingScreen-1.3.0 - 1.7.10-Forge10.13.3.1389-1710ls

**Comment by AlexIIL** [comment]
Hang on, what is the issue here? Yes this mod doesn't work with forge build 1389 specifically because forge adds its own loading screen- however forge does not have a stable API, so build 1389 for 1.7.10 will never be supported. When forge implements a proper API (I have no idea when this is happening, except that it hasn't happened yet) I will properly replace forge's loading screen with my own and follow it. Until then build 1389 is not supported

**Comment by AlexIIL** [comment]
Until then, I believe that the previous build of forge for 1.7.10 (1388) has the same features and code base- the only thing that 1389 added was forge's loading screen

**Comment by DPxKnightxZero** [comment]
Okay except the latest forge maybe a test but at this point it also fixed close 20 huge memory leaks including many fluid issues. The fps has  a 70 + boost on my pc since the latest version My pc can get as high as 330 and as low as 58 due to forge fixes. If you don't want your mod use thats fine. Forge will always be changing as modders we go in knowing that.

**Comment by AlexIIL** [comment]
Oh hang on, that changes things... I didn't think that the test version did anything else other than the loading screen. Also, I think you took me the wrong way: I don't mean to drop behind forge, however supporting a test version (and if it is not just a test version, that changes things).

**Comment by AlexIIL** [comment]
H

**Comment by AlexIIL** [comment]
I only ask because there haven't been any changes to the Minecraft Forge code between 1388 and 1389, except changes to ONLY the progress displayer and calls to it

**Comment by DPxKnightxZero** [comment]
Yes I did. I am a beta for many mods and many programs so I consider it a requirement to get my facts straight. I know its annoying and yes they did overhaul the GUI but I can't image smart people like yourself having to much issue as long as smart people like myself explain it in a way which makes sense. (Smart but mildly ADD sometimes.)

**Comment by AlexIIL** [comment]
It looks like it can be disabled via a config file in a resource-pack (that you can download into your resource pack folder) that will disable fml's loading screen. Hopefully this is just a quick fix

**Comment by AlexIIL** [comment]
Or not. My resource pack doesn't seem to be overriding fml's splash.properties file, not sure if this is me not creating a resource pack properly (very possible), or forge opening its config before any resource packs have been loaded. Either way, a fix is to navigate to ".minecraft/libraries/net/minecraftforge/forge/1.7.10-10.13.3.1389-1710ls", open the jar file with a zip opener, navigate to "assets/fml/" and open the splash.properties file, and change the line "enable = true" to "enabled = false". Its not a great fix, however it is either that or doing some really strange core-modding, just for this version.

**Comment by DPxKnightxZero** [comment]
Well ... I am not sure I think you should pm the forge guys and ask them what they did they are pretty quick to respond. I like them.

**Comment by AlexIIL** [comment]
If I had time right now I would step through the code in a dev environment and see what's happening :) Unfortunately I have a piece of A2 computing coursework due next week, and exams soon after, so I don't have the time right now :( (Apologies, I could balance revision and modding quite well during the Easter holidays, but I can't balance revision and school AND proper time for modding right now, so I am being a bit slow to update, and why I am kinda keen to wait for forge to settle down)

**Comment by DPxKnightxZero** [comment]
Hey you know what life happens. I respect modders who are real and do what they can because I am a debugger who has exactly those feelings, Just do what you can when you can ... no pressure!

**Comment by AlexIIL** [comment]
Ok, exams are finished now and forge hasn't changed their loading code for quite a while now so I will integrate properly with forge. It will be with the 1.4.x branch (completely unreleased, allows so much more customisation and future expansions) which is currently for 1.8, so a 1.7.10 release will be way after that.
